### ISSUE TITLE
Picture missing in Pipeline-Instrcutions.md

### ISSUE THREAD

**Issue by h1f1x** [aws-samples/cookiecutter-aws-sam-pipeline#1]
Pipeline-Instrcutions.md refers to a picture, which is not created when doing sam init.

How to reproduce:
```
➜ sam init --location gh:aws-samples/cookiecutter-aws-sam-pipeline
[+] Initializing project structure...
project_name [Name of the project]: t
Select source_code_repo:
1 - CodeCommit
2 - Github
Choose from 1, 2 [1]: 2
 [INFO]: Copying buildspec.yaml to current directory...
 [INFO]: Copying pipeline.yaml to current directory...
 [INFO]: Copying Pipeline-Instructions.md to current directory...
 [INFO]: Cleaning up leftovers...
 [SUCCESS]: Project initialized successfully!
[*] Project initialization is now complete

➜ tree
.
├── Pipeline-Instructions.md
├── buildspec.yaml
└── pipeline.yaml
```

**Comment by heitorlessa** [comment]
Fixed that in PR #3, just waiting for anyone to review before merging.

**Comment by heitorlessa** [comment]
Resolving as PR has been merged ;)

Thaks again @h1f1x !!
